.github/workflows/unittest.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install tox
+          sudo apt-get update
```

Comment:
Should we update every time that it runs ?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners
Github recommends this as well

Reply:
Thank you, good to know